front/src/Components/Events.js

```diff
@@ -161,6 +161,7 @@ class Events extends React.Component {
         {this.renderEventsBox()}
         {this.renderMoreRead()}
         {this.renderCreateAccount()}
+        <SearchBox />
```

Comment:
Module not found: Error: Can't resolve './SearchBox' in '/usr/local/go_appengine/gopath/src/github.com/enow-dev/enow/front/src/Components'